pkg/agent/node/node.go

```diff
@@ -32,17 +32,15 @@ func Params() map[string]interface{} {
 		"requestedHostname": os.Getenv("CATTLE_NODE_NAME"),
 	}
 
-	for k, v := range params {
-		if m, ok := v.(map[string]string); ok {
-			for k, v := range m {
-				logrus.Infof("Option %s=%s", k, v)
-			}
-		} else {
-			logrus.Infof("Option %s=%v", k, v)
+	dclient, err := client.NewClientWithOpts(client.WithAPIVersionNegotiation(), client.FromEnv)
+	if err != nil {
+		logrus.Errorf("Error getting docker client with API version negotiation, using default client from Env: %v", err)
```

Comment:
Hmm, do we still need to do fallback?

Reply:
I think so, because ClientWithOpts errors out if the first option fails https://github.com/moby/moby/blob/master/client/client.go#L133

Reply:
But client.WithAPIVersionNegotiation() doesn't return an error...

Reply:
oh, hm that's weird...so ok it'll always get a valid api version, that will work - removing the default client